packages/react/src/components/Button/Button.stories.mdx

```diff
@@ -83,3 +91,41 @@ Any actions that's less-pronounced.
 <Canvas>
   <Story name="Text" args={{children: 'Button', variant: 'text'}} />
 </Canvas>
+
+### Social
+
+Variation that provides an extra visual weight to identify the primary action from a set of buttons.
```

Comment:
We might need to update the desc to add info about disabled state :)

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/156/commits/0f9fa5b6e2dd61aa147c7aa60e68bfd1c97ec6be